docs/resources/aviatrix_smart_group.md

```diff
@@ -56,13 +56,43 @@ resource "aviatrix_smart_group" "test_smart_group_ip" {
       s2c = "remote-site-name"
     }
 
+    // Below are external group type examples
+
+    match_expressions {
+      external = "geo"
+      ext_args = {
+        country_iso_code = "US"
+        is_in_eu = "1"
+        continent_code = "NA"
+      }
+    }
+
     match_expressions {
-      external = "External_ID"
+      external = "threatiq"
       ext_args = {
-        external_ID_specific_field1 = "value1"
-        external_ID_specific_field2 = "value2"
-      } 
```

Comment:
Do you want to keep the generic example too?

Reply:
added back, thanks